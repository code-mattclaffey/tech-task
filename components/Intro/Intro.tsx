import React from "react";

// @ts-ignore
import { Heading, Text } from "@titan-tooling/ui";

type Props = {
    title: string;
};

export const Intro: React.FunctionComponent<Props> = ({ title, children }) => (
    <>
        <Heading as="h1" type="h3" additionalClassNames="u-text--align-center">
            {title}
        </Heading>
        <Text additionalClassNames="u-text--align-center">{children}</Text>
    </>

);
