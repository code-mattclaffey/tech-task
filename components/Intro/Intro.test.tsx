import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { Intro } from './Intro';

afterEach(() => {
    cleanup();
});

describe('<Intro />', () => {
    it('should match snapshot', () => {
        const { container } = render(
            <Intro title="Hello World">
                This is some jsx content
            </Intro>
        );

        expect(container).toMatchSnapshot();
    });
});
