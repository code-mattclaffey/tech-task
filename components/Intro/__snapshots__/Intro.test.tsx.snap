// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Intro /> should match snapshot 1`] = `
<div>
  <h1
    class="e-heading e-heading--h3 u-text--align-center"
  >
    Hello World
  </h1>
  <p
    class="e-text u-text--align-center"
  >
    This is some jsx content
  </p>
</div>
`;
